webapp/app.py

```diff
@@ -628,7 +628,7 @@ def takeovers_index():
 core_services_guide.init_app(app)
 
 
-app.add_url_rule("/user-country.json", view_func=get_user_country_by_ip)
+app.add_url_rule("/user-country.json", view_func=get_user_country_by_tz)
```

Comment:
If you are replacing this completely, then you might as well delete `get_user_country_by_ip` function

Reply:
Ok, but will do separately when updating the takeovers as well